ee/packages/omnichannel-services/src/OmnichannelTranscript.ts

```diff
@@ -525,7 +470,7 @@ export class OmnichannelTranscript extends ServiceClass implements IOmnichannelT
 					file: transcriptFile,
 					message: {
 						// Translate from service
-						msg: await translationService.translateToServerLanguage('pdf_success_message'),
+						msg: this.translator.t('pdf_success_message', { lng: serverLanguage, fallbackLng: 'en' }),
```

Comment:
do we need to provide `fallbackLng` here even though we've already provided at `i18n.init`?

Reply:
No need tho, i was testing and forgot this one